spec/unit/installer/analyzer/target_inspector_spec.rb

```diff
@@ -232,6 +232,8 @@ module Pod
         target_inspector = TargetInspector.new(target_definition, config.installation_root)
         platforms = target_inspector.send(:compute_platform, user_targets)
         platforms.should == Platform.new(:ios, '4.0')
+        UI.warnings.should.include 'Automatically assigning platform ios with version 4.0 on target default because no ' \
```

Comment:
Sounds good. Updated and updated specs here https://github.com/CocoaPods/cocoapods-integration-specs/pull/98